File: python_scripts/script_1_sol.py
# Script 1 Solutions
#%%  Solution Breakout 1
abspath = os.path.abspath(__file__)
dname = os.path.dirname(abspath)
os.chdir(dname + '\..')

# read in data
peach = pd.read_csv('data/peach.csv',
                   header = None)
x, y = list(peach[0]), list(peach[1])
x.append(x[0])
y.append(y[0])

# plot data
ax.fill_between(x, y, color = [0.8980, 0.3176, 0.1765]) # RGB values for orange
ax.plot(x, y, color = 'black', linewidth = 2)


#%%  Solution Breakout 2
abspath = os.path.abspath(__file__)
dname = os.path.dirname(abspath)
os.chdir(dname + '\..')

# read in data
peach = pd.read_csv('data/peach.csv',
                   header = None)
x, y = list(peach[0]), list(peach[1])
x.append(x[0])
y.append(y[0])

# plot data
ax.fill_between(x, y, color = [0.8980, 0.3176, 0.1765])
ax.plot(x, y, color = 'black', linewidth = 1)  # change to 1


#%%  Solution Breakout 3
abspath = os.path.abspath(__file__)
dname = os.path.dirname(abspath)
os.chdir(dname + '\..')

# read in data
peach = pd.read_csv('data/peach.csv',
                   header = None)
x, y = list(peach[0]), list(peach[1])
x.append(x[0])
y.append(y[0])

# plot data
ax.fill_between(x, y, color = [0.8980, 0.3176, 0.1765])
ax.plot(x, y, color = 'black', linewidth = 1)

File: matlab_scripts/script_1_sol.m
 This script has the fixed code if you are struggling to debug!

%% SOLUTION: Breakout 1 

%read in data
data = csvread('peach.csv');
%plot data
p = patch(data(:,1), data(:,2), [0.8980 0.3176 0.1765]); % RGB values for orange
p.LineWidth = 2;

%% SOLUTION: Breakout 2

%read in data
data = csvread('peach.csv');
%plot data
p = patch(data(:,1), data(:,2), [0.9882 0.1961 0.3490]); % Change color
p.LineWidth = 1; % change to 1

%% SOLUTION: Breakout 3

%read in data
data = csvread('peach.csv');
%plot data
p = patch(data(:,1), data(:,2), [0.8980 0.3176 0.1765]);  % Change color back
p.LineWidth = 1;
